README.md

```diff
@@ -56,61 +76,57 @@ A number of default values are used when installing Ansible Tower on the target
 * `tower_rabbitmq_password`: The password required to login to RabbitMQ (default password is `CHANGEME`)
 * `tower_pg_password`: The password required to login to PostgreSQL (default password is `CHANGEME`)
 
-## Bootstrapping
+### 3.1 Tower Host Name Update
+
+Once the new Tower instance has successfully been installed on the target Openshift cluster, the host name of the new Tower instance must be placed into the `tower_host` variable value, which is located in the `tower.yml` file in the `external_credentials_repo` project.
 
-### Tower Bootstrapping
+```bash
+<env>_tower_host: 'tower.example.com'
+```
 
-The `bootstrap.yml` playbook will run the Prerequisite, Integreatly and Cluster Create bootstrap playbooks in succession. These individual playbooks can be run independently if required, with instructions on how to do so in the following sections.
+## 4. Bootstrapping
 
-The bootstrap playbook requires a number of configs to be passed in, namely:
+### 4.1 Tower Bootstrapping
 
-* `tower_host`: The hostname of the target Ansible Tower instance
-* `tower_username`: The username to login with when integrating with Tower (requires admin permissions)
-* `tower_password`: The password of the tower_user specified
-* `tower_environment`: The Ansible Tower environment (dev/test/qe)
-* `tower_verify_ssl`: (Optional) Determines whether to verify the SSL certs installed on the Tower instance. By default this is set to `false`
+The `bootstrap.yml` playbook will run the Prerequisite, Integreatly, Cluster Create and Cluster Teardown bootstrap playbooks in succession. These individual playbooks can be run independently if required, with instructions on how to do so in the following sections. The playbook requires the target tower environment to be specified.
+
+* `tower_environment`: The Ansible Tower environment (dev/test/qe etc.)
 
 ```bash
-ansible-playbook -i inventories/hosts playbooks/bootstrap.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment>
+ansible-playbook -i <path-to-local-credentials-project>/inventories/hosts playbooks/bootstrap.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment> --ask-vault-pass
```

Comment:
All you need to include here is tower_environment, the creds repo will handle the [rest](https://github.com/integr8ly/tower_dummy_credentials/blob/master/inventories/group_vars/all/tower.yml)